infrastructure/simulation-tool/src/simulations/setup.ts

```diff
@@ -31,7 +31,7 @@ const depositToSelf = async (funderL2Wallet: RollupWallet, amount: BigNumber) =>
         amount
     });
 
-    return await promiseOfDeposit.awaitReceipt();
+    return await promiseOfDeposit.awaitVerifyReceipt();
```

Comment:
do we need to wait for the verification? it's just that if not then it's unnecessarily slow